db/migrate/20170110090935_add_ems_to_miq_alert_status.rb

```diff
@@ -0,0 +1,20 @@
+class AddEmsToMiqAlertStatus < ActiveRecord::Migration[5.0]
+  class MiqAlertStatus < ActiveRecord::Base; end
+
+  def up
+    add_column :miq_alert_statuses, :ems_id, :bigint
```

Comment:
@moolitayer  can you do structural changes(you can use `change` method) in one migration and data changes in other, please ?